sp800-63d/vot_mapping.md

```diff
@@ -34,38 +36,70 @@ IAL maps into the VoT component P using a numeric identifier. Only one numeric i
 |IAL2|P2|
 |IAL3|P3|
 
-In addition, the type of proofing used MAY be indicated using an alphabetic identifier. Multiple alphabetic identifiers MAY be used simultaneously.
+#### Identity Proofing Characteristics
+
+In addition, identity proofing characteristics MAY be indicated using an alphabetic identifier. Multiple alphabetic identifiers MAY be used simultaneously. Identity proofing methods or approaches that are required for the asserted IAL SHOULD NOT be asserted.
 
-|Proofing method|Component Value|
-|In-person|Pp|
+|Identity Proofing Method or Approach|Component Value|
+|:----:|:--:|
+|In-person|Pi|
 |Remote|Pr|
+|Knowledge-based Verification (KBV)|Pk|
+|Address Confirmation with Postal Address|Pa|
+|Trusted Referee|Pt|
+|Implemented IAL differs from assessed IAL|Px|
 
-### AAL to VOT Component Values
+>Note: Any credential service provider performing identity proofing at IAL2 or IAL3 MUST conduct a privacy risk assessment. Trusted referee policy and procedures MUST be documented per SP 800-63A Section 5.3.4.

+If Px is asserted, the component value for the assessed IAL MUST also be asserted (e.g., “P2.Px”). The discrepancy in assessed and implemented assurance level SHOULD be documented in a digital identity acceptance statement, per SP 800-63-3 Section 5.5.
```

Comment:
The numeric level should be a *floor* of the vector component level, any alphabetic bits should satisfy or be additive to the numeric component, not removing capability from it. So instead of P2.Px, you would have P1.Pt.Pk or something like that. An RP can then decide "I'll take any P2 or a P1 only if it has Pt and Pk as well" for a particular action. 

Also, the numeric value is required at all times, whether or not there is an exception. I would recommend that instead of a generic "exception" flag we be more specific, where possible, what the exceptions are and be clear that any exceptions do not remove from the base level definitions.